src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,20 @@
+package duke.tasks;
+
+/**
+ * Represents a Todo object that is a Task.
+ */
+public class Todo extends Task {
```

Comment:
A suggestion is if you would like to give Todo tasks date functionality as well

Reply:
But would it not make Todo more like the Deadline Task?